src/image-carousel/image-carousel.style.tsx

```diff
@@ -0,0 +1,196 @@
+        width: 1.5rem;
+    }
+    ${(props) =>
+        props.position === "left" &&
+        css`
+            left: 3rem;
+            ${MediaQuery.MaxWidth.tablet} {
+                left: 2rem;
+            }
+            ${MediaQuery.MaxWidth.mobileL} {
+                left: 1.25rem;
+            }
+        `}
+
+    ${(props) =>
+        props.position === "right" &&
+        css`
+            right: 3rem;
+            ${MediaQuery.MaxWidth.tablet} {
+                right: 2rem;
+            }
+            ${MediaQuery.MaxWidth.mobileL} {
+                right: 1.25rem;
+            }
+        `}
+`;
+export const ImageGalleryContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+`;
+export const ImageGalleryWrapper = styled.div`
+    user-select: none;
+    position: relative;
+    display: block;
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+`;
+
+export const ImageGallerySwipe = styled.div`
+    display: block;
+    user-select: none;
+    height: 100%;
+    overflow: hidden;
+    touch-action: pan-y;
+`;
+export const ImageGallerySlides = styled.div`
+    display: flex;
+    transition: all 450ms ease-out 0s;
+    position: relative;
+    text-align: center;
+    user-select: none;
+    width: 100%;
+    height: 100%;
+`;
+export const ImageGallerySlide = styled.div`
+    flex: 0 0 100%;
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    .react-transform-wrapper {
+        height: 100%;
+    }
+    .react-transform-component {
+        height: 100%;
+    }
+`;
+
+export const ImageBox = styled.img`
+    max-height: 100%;
+    max-width: 100%;
+    margin: auto;
+    height: auto;
+    width: auto;
+    object-fit: contain;
+`;
+export const BoxChip = styled.div`
+    display: flex;
+    justify-content: center;
+    padding: 1rem;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    margin-left: auto;
+    margin-right: auto;
+    width: 100px; /* Need a specific value to work */
+    z-index: 3;
+`;
+export const Chip = styled.div`
+    display: inline-flex;
+    padding: 4px 16px;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+    border-radius: 20px;
+    background-color: ${Color.Neutral[8]};
+    text-align: center;
+    font-weight: 600;
+    font-size: 0.75rem;
+    line-height: 20px;
+    letter-spacing: 0.12px;
+`;
+
+export const ThumbnailContainer = styled.div`
+    display: flex;
+    overflow: auto;
+    padding: 1rem 0.625rem;
+    background-color: ${Color.Neutral[1]};
+`;
+export const ThumbnailWrapper = styled.div`
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    justify-content: center;
+`;
+export const ThumbnailItem = styled.div<{
+    acctive?: boolean;
+}>`
+    cursor: pointer;
+    background-color: ${Color.Neutral[1]};
+    border-radius: 10px;
+    height: 4.24rem;
+    width: 4.24rem;
```

Comment:
based on Figma the thumbnail dimensions should be 100px => 6.25rem on desktop and 64px => 4rem on mobile